website/docs/core-concepts/connecting-to-data-sources/authentication/self-signed-certificates.md

```diff
@@ -1,6 +1,6 @@
-# Self-signed Certificates
+# Self Signed Certificates
 
-Self-signed certificates are sometimes used in testing or internal environments, as they provide incoming/outgoing data encryption and can be created quickly with no cost. This practice can carry significantly increased security risks, however, and should be carefully considered.
+Self-signed certificates are sometimes used in testing or internal environments, as they provide incoming/outgoing data encryption and can be created with no cost. It's important to consider the potential security risks associated with this practice, as they may carry significantly increased risks.
```

Comment:
Sorry, I could not understand this line - `It's important to consider the potential security risks associated with this practice, as they may carry significantly increased risks.`

Reply:
removed the para